### DISCUSSION TITLE
Auswahlfelder & + Hinzufügen

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#23] [Shipped - Live on production]
Text in den Auswahlfeldern (Bspw. "Gerätetyp auswählen" bei Inventar anlegen) umbenennen in

"xyz suchen & auswählen"

"+ Hinzufügen" verleitet aktuell dazu zu schnell einen neuen Typen oder ähnliches anzulegen

Vorschlag: 
1. "+ Hinzufügen" Button erst nach Eingabe von min einem Suchwert anzeigen
2. Spezifisch umbenennen und ggf. kursiv oder als blauen Button damit es sich mehr abhebt? "_xyz nicht gefunden? + Neu hinzufügen_"

![Bildschirmfoto 2022-08-10 um 15 47 53](https://user-images.githubusercontent.com/80519295/183918496-ace62dd1-c855-47db-a92c-31a6e79a3045.png)

**Comment by YvesR** [comment]
Small UX: Labels anpassen
